templates/vault/0/rancher-compose.yml

```diff
@@ -0,0 +1,120 @@
+version: '2'
+catalog:
+  name: Vault
+  version: 1.0-rancher1
+  description: |
+    Installs a Vault container, internal (Rancher) load-balancer, and allows connection to a Consul cluster
+  uuid: vault-rac-0
+  questions:
+    - variable: VAULT_CLUSTER_NAME
+      label: Vault Cluster Name
+      description: |
+        Provide a name for this Vault cluster
+      type: string
+      default: myCluster
+      required: true
+    - variable: VAULT_LISTEN_PORT
+      label: Vault Listen Port
+      description: |
+        This is the port number you want Vault to listen on.  Actually, Vault ALWAYS listens on port 8200, but it isn't exposed, so you can set the port you want the LoadBalancer to listen on.
+      default: 8200
+      type: int
+      required: true
+    - variable: VAULT_CLUSTER_PORT
+      label: Vault Cluster Port
+      description: |
+        Similar to the Vault Listen Port, this is the VAULT CLUSTER listen port to use.
+      default: 8201
+      type: int
+      required: true
+    - variable: VAULT_BACKEND
+      label: Select a backend for Vault
+      description: |
+        Vault supports several different backend storage engines.  Please select the one you are interested in using. (NOTE: Must be pre-installed or added to a different stack)
+      default: Filesystem
+      type: enum
+      required: true
+      options:
+        - azure
+        - cockroachdb
+        - consul
+        - couchdb
+        - dynamodb
+        - etcd
+        - file
+        - gcs
+        - inmem
+        - manta
+        - mysql
+        - postgresql
+        - spanner
+        - cassandra
+        - s3
+        - swift
+        - zookeeper
+    - variable: BACKEND_SERVICE
+      label: Specify the Backend service to connect to
+      description: |
+        The pre-installed Backend server to back to.  To reference, you can use the memory variable "SERVICE" in your URLs.  For example: "consul://SERVICE:8500"
+      default: "select"
+      type: service
+    - variable: BACKEND_CONFIGURATION
+      label: Specify the backend configuration in valid JSON format
+      default: |
+        "path":"/mnt/vault/data"

```

Comment:
Correct path as default should be same as volume you create at `docker-compose.yml.tpl`, added in last review.
```
-        "path":"/mnt/vault/data"
+        "path":"/vault/file"
```